docs/en/enterprise-edition/policy-reference/docker-policies/docker-policy-index/ensure-that-a-user-for-the-container-has-been-created.adoc

```diff
@@ -28,9 +28,7 @@
 === Description 
 
 
-Containers should run as a non-root user.
-It is good practice to run the container as a non-root user, where possible.
-This can be done either via the `USER` directive in the `Dockerfile` or through `gosu` or similar where used as part of the `CMD` or `ENTRYPOINT` directives.
+The policy's primary purpose is to verify that a user has been explicitly created for running the container. This practice is crucial because it avoids running the container with root privileges, which could pose significant security risks if the container were compromised. Running containers as a non-root user reduces the potential impact of a breach.
```

Comment:
The policy's primary purpose is to verify that a dedicated user has been explicitly created for running the container. This is essential to avoid running the container with root privileges, which could introduce significant security risks in case of a compromise. Running containers as a non-root user reduces the potential impact of a security breach.